src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -968,6 +1005,7 @@ registerThemingParticipant((theme, collector) => {
 	const tileBackgroundColor = theme.getColor(inputBackground);
 	if (tileBackgroundColor) {
 		collector.addRule(`.monaco-workbench .part.editor > .content .welcomePageContainer .tile:not(.extension):not(.extension-pack) { background-color: ${tileBackgroundColor};  }`);
+		collector.addRule(`.monaco-workbench .part.editor > .content .welcomePageContainer .btn-secondary .monaco-button { background-color: ${tileBackgroundColor} !important;  }`);
```

Comment:
Why is the !important needed here? Is there something else setting the color you need to override?

Reply:
Hey Charles, yeah i didnt want to do that. I was hoping to get some feedback. This is to override the inline color that are set on the monaco button. Any suggestions?

Reply:
I'd suggest looking into calling button.style directly - e.g. https://github.com/microsoft/azuredatastudio/blob/main/src/vs/base/browser/ui/dialog/dialog.ts#L295